SunGather/registers.yaml

```diff
@@ -1140,14 +1141,14 @@ registers:
       accuracy: 0.1
       unit: "kWh"
       models: ["SH5K-20","SH3K6","SH4K6","SH5K-V13","SH5K-30","SH3K6-30","SH4K6-30","SH5.0RS","SH3.6RS","SH4.6RS","SH6.0RS","SH10RT","SH8.0RT","SH6.0RT","SH5.0RT"] 
-    - name: "load_power_hybrid"    
+    - name: "load_power"
```

Comment:
changing the name will break any existing queries/dashboards etc. that rely on it